packages/wdio-utils/src/test-framework/testFnWrapper.ts

```diff
@@ -8,11 +8,18 @@ import type {
     AfterHookParam
 } from './types.js'
 
+declare global {
+    // Firstly variable '_wdioDynamicJasmineResultErrorList' gets reference to test result in packages/wdio-jasmine-framework/src/index.ts and then used here in wdio-utils/ as workaround for Jasmine
+    // eslint-disable-next-line no-var
+    var _wdioDynamicJasmineResultErrorList: any
```

Comment:
The global is only defined in Jasmine
```suggestion
    var _wdioDynamicJasmineResultErrorList?: any
```